_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "NewPipe 0.19.0 released: Folder your feeds and mute them"
+short: "NewPipe 0.19.0 released"
+date: 2020-03-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a great way to sort your subscriptions and a mute button, on top of the usual smaller improvements and fixes<!-- more -->
+
+
+## Folders
+
+Since moving images say more then my words, please have a look at them:
+
+<video src="/img/folders.mp4" class="no-flow img-responsive" width="220" height="140" autoplay loop></video>
+
+For everyone who rather enjoys reading, this is for you: NewPipe now supports grouping subscriptions into folders. All the recent videos of those channels then appear in one re-loadable feed when you press on the folder. While [@mauriciocolli](https://github.com/mauriciocolli) implemented that, he also improved the `What's New` sidebar feed. And on top of that, live streams have now a viewer count in those feeds!
```

Comment:
For everyone who rather enjoys reading->For those who prefer to read

The other wording sounds.... off

Reply:
I don't prefer any version, so will enjoy using yours.